### DISCUSSION TITLE
ZEPETOScript에서 Vector3 Operator를 사용할 수 있나요?

### DISCUSSION THREAD

**Discussion by mayko1234** [naverz/zepeto-studio-kor#295] [5. Q&A]
안녕하세요, 제페토스크립트로 Vector3 계산을 하려고 하는데 안되네요. 
아직 구현이 안된건지, 제가 뭘 잘 못하고 있는 건지 모르겠네요. 혹시 아시는 분 있으면 답변 부탁드립니다. 
![image](https://user-images.githubusercontent.com/97988822/151530855-292c5cf3-ef8b-4273-8523-05821a8f7f0f.png)

**Comment by runasy-koonta** [comment]
VS Code의 코드 검증 기능이 이상한 것 같더라고요,
Jetbrains Rider에서는 문제 발생하지 않습니다.

빨간 줄 떠도 문제없이 컴파일되니 무시하셔도 됩니다.

추가)
VS code에서 Ctrl + Shift + P를 눌러서 나오는 입력 창에서, TypeScript: Select Typescript Version 메뉴를 선택한 뒤 Use Workspace Version을 사용하면 해결됩니다.

기본값이 VS Code에 내장된 typescript인 것 같습니다.
Use workspace version을 선택하면 .tsconfig에 지정된 대로, zepeto-script-compiler plugin을 불러와 Custom Operator를 사용할 수 있습니다.
관련 자료는 node_modules/zepeto-script-compiler/README.md를 열어보시면 확인할 수 있습니다.

**Reply by mayko1234** [reply]
헐.. README에 적혀있네요. 왜 이렇게 찾아볼 생각을 못했을까요?
솔루션을 찾아서 기쁨+왜 혼자서는 못찾았지 하는 자괴감이..ㅠㅠ 
얼마전에 Rider가 무료 체험이 끝나서 VS Code로 바꾼 참이였거든요. 
말씀해주신 솔루션 대로 하니까 빨간 줄이 사라졌네요 ㅎㅎ
답변 넘 감사합니다! 꾸벅~

**Reply by x4cf** [reply]
와우! 좋은 정보 얻어가요 :D
